hbase-shell/src/main/ruby/hbase/admin.rb

```diff
@@ -1137,7 +1137,7 @@ def cfd(arg, tdb)
         algorithm = arg.delete(ColumnFamilyDescriptorBuilder::ENCRYPTION).upcase
         cfdb.setEncryptionType(algorithm)
         if arg.include?(ColumnFamilyDescriptorBuilder::ENCRYPTION_KEY)
-          key = org.apache.hadoop.hbase.io.crypto.Encryption.pbkdf128(
+          key = org.apache.hadoop.hbase.io.crypto.Encryption.pbkdf384(
```

Comment:
Why are we chaninging the default here? If this needs to be changed please open an new issue to discuss.

Reply:
OK, I'll revert this part. I originally added this change to eliminate SHA1 algorithm use (in the pbkdf128 method). 

Here we use a key generation algorithm to generate a secret key (based on the string provided in the hbase shell) which will be used to encrypt the store/wal files. According to HBASE-10951 (where this algorithm was already changed once), this is only used for testing schema in the shell, the proper secret keys should be generated by using the Java API. 

However, I'm not 100% sure if someone is not using this method in the HBase shell in production now to define the secret key, so maybe it is not a good idea to change. I agree that it worths further discussion.